desktop/core/src/desktop/js/parse/sql/generic/jison/select/select.test.json

```diff
@@ -3927,6 +3927,70 @@
       "lowerCase": false
     }
   },
+  {
```

Comment:
Next three tests are similar to the ones above for `CAST(foo AS type)` expressions